src/triggered/UpdateSubstation.ts

```diff
@@ -0,0 +1,83 @@
+import { css, html, LitElement, query, TemplateResult } from 'lit-element';
+import { get } from 'lit-translate';
+
+import { newWizardEvent } from '../foundation.js';
+import { Diff, mergeWizard } from '../wizards.js';
+
+function lNSelector(lNode: Element): string {
+  const [iedName, ldInst, prefix, lnClass, lnInst] = [
+    'iedName',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195000)
```suggestion
    "iedName",
```